cmd/operator_install.go

```diff
@@ -83,8 +83,11 @@ func operatorInstall(config *operatorInstallCommandConfig) error {
 		if existingOperatorWatchspace == "" {
 			existingOperatorWatchspace = "all namespaces"
 		}
-		if existingOperatorWatchspace != operatorNamespace {
-			return errors.Errorf("An Appsody operator already exists in namespace %s but is watching another namespace: %s. Contact your cluster administrator for more details.", operatorNamespace, existingOperatorWatchspace)
+		watchSpaces := getWatchSpaces(existingOperatorWatchspace, config.Dryrun)
+		for _, existingWatchspace := range watchSpaces {
+			if existingWatchspace != operatorNamespace {
```

Comment:
I think the logic needs to be adjusted here. We need to error out only if none of the `existingWatchspace` match the operatorWatchspace. Say the operator is installed in `xyz`, and it watches `abc`, `def`, and `xyz`. The code would error out on `abc` != `xyz`, but it shouldn't because the operator does watch `xyz`